extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -105,6 +105,7 @@ export let InteractiveComponentDef;
  *    currentPageIndex: number,
  *    pagesCount: number,
  *    pageIds: !Array<string>,
+ *    newPageAvailableId: string,
```

Comment:
Naming nits: could you please find a consistent name across the store property and action, the system layer attribute and class?

My point is that it might be hard for a new developer to figure out that `HAS_NEW_UPDATE_ATTRIBUTE` in the system layer means that there's a new page from the livelist. An update could technically be anything (consent, access, subscriptions, livelist, reactions, a video loaded ...)

Reply:
That's a good point 👍 Done.